phases/phase-1/challenges/likes/lixa4ev2010

```diff
@@ -0,0 +1,2 @@
+C3A79AE2A7BF950AD4DAF49629DAEDFAB5BE2F62548F943679AB941F8576F101
+C3A79AE2A7BF950AD4DAF49629DAEDFAB5BE2F62548F943679AB941F8576F101
```

Comment:
Could you please remove this as it's duplicated?

Reply:
Done